feat: client docs (#19)
ipfs-shipyard/nft.storage: client/README.md

# nft.storage

> A client library for https://nft.storage/ service. It provides convenient interface for working with [RAW HTTP API][] from web browser or [nodejs][] and comes bundled with TS for out-of-the box type inference and better IntelliSense.

Reviewer: ```suggestion
A client library for the https://nft.storage/ service. It provides a convenient interface for working with the [Raw HTTP API][] from a web browser or [Node.js][] and comes bundled with TS for out-of-the box type inference and better IntelliSense.
```

## Install

> Install the package from npm using using npm

Reviewer: ```suggestion
Install the package using npm
```

```
npm install nft.storage
```

Or yarn

```
yarn add nft.storage
```

> ### Usage

Reviewer: ```suggestion
### Usage

First, obtain an API token from https://nft.storage and use it in place of `API_TOKEN` below:
```

```js
import { NFTStorage } from "nft.storage"
const client = new NFTStorage({ token: API_TOKEN }) 
const cid = await client.storeBlob(new Blob(['hello world'])) 
```

For more examples please see [API documentation][]

> [RAW HTTP API]:https://nft.storage/#api-docs

Reviewer: ```suggestion
[Raw HTTP API]:https://nft.storage/#api-docs
```

> [nodejs]:https://nodejs.org/

Reviewer: ```suggestion
[Node.js]:https://nodejs.org/
```
